src/how-tos/mqtt/mqtt_kafka_broker_zilla.yaml

```diff
@@ -37,9 +78,10 @@ bindings:
       bootstrap:
         - mqtt-messages
         - mqtt-retained
+        - mqtt-places
```

Comment:
`mqtt-places` or `mqtt-devices`?

Reply:
mqtt-devices, this was copied from the example.